portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -46,6 +46,17 @@ if (end > total) {
 String deltaURL = HttpUtil.removeParameter(url, namespace + deltaParam);
 
 NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
+
+if (forcePost && (portletURL != null) && Validator.isNotNull(url)) {
```

Comment:
@brunobasto i just need portletURL because of taglib on line 55. In this stretch we can only use portletURL. But we need remove all parameter if we'll use on line 54. 

Thx.